spec/spec-suite-jasmine.js

```diff
@@ -0,0 +1,8 @@
+'use babel'
+/* @flow */
+
+require('./spec-suite')
+
+module.exports = function () {
```

Comment:
Just curious, any reason this is using `function ()` where other functions in this PR use `() =>`?